ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -1872,14 +1875,108 @@ require(['jquery', 'angular', 'JobRunner', 'angular-resource', 'angular-ui-boots
 
   driveDirectives.directive('fileUpload', ['$upload', function($upload) {
     var formToken = $('meta[name=form_token]').attr('content');
+    const uploadActions = {
+      NEW: 'newFile',
+      REPLACE: 'replace',
+      ABORT: 'abort'
+    };
+    // Used for managing file modals that need to be successively displayed.
+    var modalData = {
+      'index': 0,
+      'repeatAction': null,
+      'modals': []
+    };
```

Comment:
What is the benefit of grouping ``repeatAction`` and ``modals``? You don't seem to pass ``modalData`` as a function argument. And the naming could be improved. For instance ``modals`` is misleading because it doesn't contain any actual modal but some data used by modals.